src/main/java/ee/carlrobert/codegpt/completions/HuggingFaceModel.java

```diff
@@ -59,7 +59,30 @@ public enum HuggingFaceModel {
   PHI_3_3_8B_4K_Q5_K_S(4, 5, "Phi-3-mini-4k-instruct-Q5_K_S.gguf", "lmstudio-community", 2.82),
   PHI_3_3_8B_4K_Q6_K(4, 6, "Phi-3-mini-4k-instruct-Q6_K.gguf", "lmstudio-community", 3.14),
   PHI_3_3_8B_4K_Q8_0(4, 8, "Phi-3-mini-4k-instruct-Q8_0.gguf", "lmstudio-community", 4.06),
-  PHI_3_3_8B_4K_FP16(4, 16, "Phi-3-mini-4k-instruct-fp16.gguf", "lmstudio-community", 7.64);
+  PHI_3_3_8B_4K_FP16(4, 16, "Phi-3-mini-4k-instruct-fp16.gguf", "lmstudio-community", 7.64),
+
+  CODE_QWEN_1_5_7B_Q2_K(7, 2, "CodeQwen1.5-7B-Chat.Q2_K.gguf", "RichardErkhov", 3.05),
```

Comment:
I checked with @carlrobertoh and we agreed there is no benefit in listing all available quantizations here, for https://github.com/carlrobertoh/CodeGPT/pull/529/files we agreed that it is enough to only offer  `K_M` quantizations for 3-8 Bit (if there is no `K_M`, use only `K` or `M` ones)

Reply:
So I should also drop Q2_K?